src/components/tab-group/tab/bl-tab.css

```diff
@@ -0,0 +1,152 @@
+:host {
+  position: relative;
+  display: flex;
+  align-items: center;
+  background-color: var(--bl-color-primary-background);
+
+  --bl-tab-title-padding-vertical: var(--bl-size-m);
+  --bl-tab-title-padding-horizontal: var(--bl-size-xl);
+  --bl-tab-title-color: var(--bl-color-secondary);
+  --bl-tab-caption-color: var(--bl-color-secondary);
+  --bl-tab-icon-color: var(--bl-color-secondary);
+  --bl-tab-border-bottom-width: var(--bl-size-4xs);
+  --bl-tab-border-left-space: var(--bl-size-xl);
```

Comment:
For the same reasons that we had in badge component in #188 we need to move these _private_ variables to the wrapping element (`.container`). We can also remove `bl-tab` prefixes since these are only valid inside the component.

Reply:
This will require some refactor. We can look at together if you want.